lib/tds.js

```diff
@@ -190,17 +196,19 @@ async function generateDNRRulesForTrackerEntry (
     priority, isRegexSupported, surrogatePathPrefix, supportedSurrogateScripts
 ) {
     const dnrRules = []
+    const customActions = []
 
     if (priority > MAXIMUM_SUBDOMAIN_PRIORITY) {
         throw new Error('Too many tracker entries for domain: ' + trackerDomain)
     }
 
     const defaultAction = normalizeAction(trackerEntry.default)
-
+    const customDefaultAction = supportedCustomActions[defaultAction]
```

Comment:
Wait did we?  I honestly don't remember that and didn't see that in the TD either.  FWIW we just discussed this in https://app.asana.com/0/1171671347221384/1203389689265368 and sounds like we'll handle this just in rules. So I'll remove this for now as its not strictly needed, but seems like it introduces a bit of an odd inconsistency?

Reply:
Well, my understanding was that we did. The logic was that tracker rules with an unknown action are skipped, but default actions are always standard (block/ignore). That's since if a tracker rule is skipped, matching can fall back to other rules for the tracker, or the default action. But falling back to an unknown default action isn't going to work.